### PR TITLE
Update wechat from 2.3.31.17 to 2.3.31.18

### PR DESCRIPTION
**Pull request by huxuan**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Duplicate.

**Comment by huxuan** [comment]
Hi @vitorgalvao, may I have a small suggestion on the duplication?

I noticed that there are many duplicate pull requests with same changes, but maybe it is better the merge the earliest one rather than the latest one. Actually, every time before I send a pull request, I will double check whether there is already a similar one exists. We should not encourage this kind of duplication work.

**Comment by vitorgalvao** [comment]
> maybe it is better the merge the earliest one rather than the latest one.

I agree. But Github’s Notifications and Pull Request screens don’t make that easy, by putting the latest first.

We get inundated with so many requests, they’re hard to parse. I merged the earliest one *I could see* and then found more in the rubble.

Sometimes contributors notice the duplicates and comment with a link back to the original. That helps merge the earliest one.

**Comment by huxuan** [comment]
Hi @vitorgalvao, thanks for your reply. I totally understand and really appreciate your efforts to maintain the awesome homebrew cask.
